src/main/java/domain/Car.java

```diff
@@ -1,15 +1,43 @@
 package domain;
 
-
 public class Car {
 
-    private final Name name;
+    private static final int MOVING_STANDARD_NUM = 4;
+    private static final int ADD_POSITION = 1;
+    private static final int INIT_POSITION = 0;
+    private static final int MIN_NAME_LENGTH = 1;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
+
+    private int position = INIT_POSITION;
 
-    public Car(Name name) {
+    public Car(String name) {
+        validateNameLength(name.length());
         this.name = name;
     }
 
+    public void move(int number) {
+        if (isAllowedToMove(number)) {
+            this.position += ADD_POSITION;
+        }
+    }
+
+    private void validateNameLength(int nameSize) {
+        if (nameSize < MIN_NAME_LENGTH || nameSize > MAX_NAME_LENGTH) {
+            throw new IllegalArgumentException("[ERROR] 1에서 5사이의 이름 길이만 입력 가능합니다.");
+        }
+    }
```

Comment:
```suggestion

    private void validateNameLength(int nameLength) {
        if (!isValidNameLenght(nameLength)) {
            throw new IllegalArgumentException("[ERROR] 1에서 5사이의 이름 길이만 입력 가능합니다.");
        }
    }

    private static boolean isValidNameLenght(int nameLength) {
        return nameLength >= MIN_NAME_LENGTH && nameLength <= MAX_NAME_LENGTH;
    }

```
로 추상화를 진행해주신다면, 쉽게 읽을 수 있을 것 같아요

ex) 
기존> NameLength를 검사하네  > 1미만 5초과이면  예외를 뱉는구나.
변경 > NameLength를 검사하네 > 유효한 크기의 이름이 아니면, 예외를 뱉는구나, > 유효한 크기의 이름은 어떤 기준이지? 아~~  1이상 5이하가 유효한 크기의 이름이구나.

읽는 사람에 있어, 
valdate 기준은 구현 세부사항일 수 있어서, 적절한 이름으로 작성해주신다면, 필요할 때, 해당 기준을 살펴볼 수 있게 할 수 있을 것 같습니다 :)